utilities/feature_space_measures.py

```diff
@@ -0,0 +1,354 @@
+import numpy as np
+from utilities.scalers import NormalizeScaler, KernelNormalizer
+
+# The code is based on methods presented in the paper TODO(currently in
+# review process, will be added as soon as published)
+
+# default cross validation parameters used for the paper
+DEFAULT_CROSS_VALIDATION_KWARGS = {
+    "train_test_split": True, "train_ratio": 0.5, "seed": 0x5F3759DF, "nb_folds": 2}
+
+
+def standardize_features(features, train_idx=None):
```

Comment:
This wrapper is not necessary based upon the amount of code within it. It would be better to directly use the scaler, fit it to the training features, and then store it for future transformations.